LICENSE.MD

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
more significantly, maybe we should use ASF2.0... not like it really matters for a docs site, but it's what the CNCF prefers and I don't have any personal preference.